docs/prow/prow-installation-on-forks.md

```diff
@@ -74,9 +74,9 @@ For the purpose of the installation, you must have a set of service accounts and
     - Compute OS Admin Login (`roles/compute.osAdminLogin`)
     - Service Account User (`roles/iam.serviceAccountUser`)
  - **sa-gcs-plank** with the role that allows the account to store objects in a bucket:
-    - Storage Object Admin (`roles/storage.objectAdmin`)
+    - Storage Object Admin (`roles/storage.objectAdmin`) in the `kyma-prow-logs` bucket
  - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project:
-    - Storage Admin (`roles/storage.admin`)
```

Comment:
It looks that there is a duplication in the documentation: https://github.com/kyma-project/test-infra/blob/master/docs/prow/authorization.md
@klaudiagrz, @piotrmiskiewicz can you fix it?

Reply:
the docs are redesigned